eZ/Publish/Core/Repository/Values/ContentType/FieldDefinition.php

```diff
@@ -116,7 +116,7 @@ public function getDescriptions()
      */
     public function getDescription($languageCode)
     {
-        if (array_key_exists($languageCode, $this->descriptions)) {
+        if (is_array($this->descriptions) && array_key_exists($languageCode, $this->descriptions)) {
```

Comment:
I had the same question as names are the same but as I don't master FielDefinition I did the less breaker way.

What I can do to make something cleaner is : 

- add a defaults variable to ValueObject
- use it in the construct to set default properties
- add name and description default to FieldDescription

Would you prefer that ?

/cc @bdunogier @andrerom